src/flags/orgFlags.ts

```diff
@@ -10,7 +10,14 @@ import { Messages, Org, ConfigAggregator } from '@salesforce/core';
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/sf-plugins-core', 'messages');
 
-const maybeGetOrg = async (input?: string): Promise<Org | undefined> => Org.create({ aliasOrUsername: input });
+const maybeGetOrg = async (input?: string): Promise<Org | undefined> => {
```

Comment:
How about some tests for all the org flags?

Reply:
I tried a few ways and hated all of them.  Open to suggestions or contributions